app/_src/policies/meshfaultinjection.md

```diff
@@ -34,14 +34,14 @@ Do **not** combine with [FaultInjection](/docs/{{ page.release }}/policies/fault
 {% endif_version %}
 {% endtab %}
 
-{% tab targetRef27x Builtin Gateway %}
+{% tab targetRef Builtin Gateway %}
```

Comment:
After https://github.com/kumahq/kuma-website/pull/2216 there shouldn't be the "id" of the tabs anymore, just the tab name:
```suggestion
{% tab Builtin Gateway %}
```